packages/formatter-html/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "description": "A hint formatter that outputs the issues in a HTML file.",
+  "dependencies": {
+    "ejs": "^2.6.1",
+    "fs-extra": "^7.0.0",
```

Comment:
Maybe replace with [`mkdirp`](https://www.npmjs.com/package/mkdirp) so we have less dependencies and smaller package?

Reply:
I'm using more than the mkdirp functionality.